docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -74,8 +74,7 @@ Setting up your environment to build documentation locally
 
 To build documentation locally, ensure you have a working :ref:`development environment <environment_setup>`.
 
-To work with documentation on your local machine, you need to have python-3.5 or greater and the
-following packages installed:
+To work with documentation on your local machine, you need to have python-3.5 or greater and the following packages installed:
```

Comment:
There was a comment in an older PR that suggests we just remove this list and say  something like see xx requirements.txt for the packages and versions required (maybe even link to that file in github.)

Reply:
thanks, I've updated based on this comment